### PR TITLE
task-05: update netdev link in README.md

### PR DESCRIPTION
**Pull request by cgundogan**
fix netdev api url

### CODE REVIEW

File: task-05/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Task 5: Using network devices
 
 ## RIOT's Networking architecture
-Network devices are accessed through [`netdev`](http://doc.riot-os.org/group__drivers__netdev__netdev2.html) driver API
+Network devices are accessed through [`netdev`](http://doc.riot-os.org/group__drivers__netdev__api.html) driver API
```
Comment (miri64) [inline task-05/README.md:6]: https

File: task-05/README.md [hunk 2]
```diff
@@ -41,6 +41,6 @@ USEMODULE += auto_init_gnrc_netif
 * Type `ifconfig` to get your hardware addresses
 * Use `txtsnd` to send one of your neighbors a friendly message
 
-[Read the Doc](http://doc.riot-os.org/group__drivers__netdev__netdev2.html)
+[Read the Doc](http://doc.riot-os.org/group__drivers__netdev__api.html)
```
Comment (miri64) [inline task-05/README.md:44]: https

**Review by miri64 — APPROVED** [review]
Please provide a follow-up ;-)

File: task-05/README.md (hunk 2 above)
Comment (cgundogan) [reply task-05/README.md:44]: so you want that as a follow-up? (:

Comment (miri64) [reply task-05/README.md:44]: Japp, if you have time for that :-)

**Pull request merged**